package.json

```diff
@@ -22,7 +22,7 @@
     "babel-preset-stage-2": "6.24.1",
     "banner-footer-webpack-plugin": "git://github.com/dtothefp/banner-footer-webpack-plugin.git",
     "browser-sync": "2.18.7",
-    "capital-framework": "3.6.1",
+    "capital-framework": "4.8.1",
```

Comment:
I'll give this a more thorough review tomorrow, but we want to switch to requiring individual CF components at this point, to make single component updates possible and hopefully prevent getting as far behind as we are now in the future.